src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const columns = document.querySelectorAll('tr');
```

Comment:
The variable 'columns' is misleading since it's actually selecting rows ('tr' elements), not columns. Consider renaming it to 'rows' to accurately reflect the elements being selected.